canonical-web-and-design/canonical.com
# 1531 rebuild partners section for mobile

## Done

- Rebuild partners section to suit mobile design

## QA
- Go to [http://0.0.0.0:8002](http://0.0.0.0:8002) 
- Compare to [design](https://github.com/canonical-web-and-design/web-squad/issues/1531)
- Make sure the links on big screen are functional
- Make sure the select navigation on mobile version is functional

## Issue

Fixes https://github.com/canonical-web-and-design/web-squad/issues/1531
